src/alert/alert.js

```diff
@@ -0,0 +1,17 @@
+import jQuery from 'jquery';
+import template from './alert.pug';
+
+export default function alert(message) {
+	const html = template({message : message});
+	const component = jQuery(html);
+
+	component.appendTo('body');
+
+	component.modal('show');
+
+	component.find('button').click(function() {
+		component.modal('hide');
+		component.remove();
+
+	});
+}
```

Comment:
configure your editor so it always leaves a "newline" character at the end of the file.

Read more about this here:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline